types/vega/index.d.ts

```diff
@@ -524,8 +524,11 @@ declare namespace vg {
     mult?: number;
     offset?: number;
     band?: boolean;
+    test?: string;
   }
 
+  export type ValueRefs = ValueRef | ValueRef[];
```

Comment:
I'm having trouble deciding a good name for the extra type

Reply:
`ValueRefs` seems fine to me. Is it really the case that all aesthetics can take an array of value refs, not just a single one? (I haven't used vega in a while).

Reply:
Seems so; tried: `x`, `y2`, `width`, `fill`, `fillOpacity`, `stroke`

Reply:
The first value in the array that has no test or that has a passing test - is used